### ISSUE TITLE
Discord?

### ISSUE THREAD

**Issue by alelordelo** [sail-sg/EditAnything#6]
Suggestion: a Discord server of EditAnything so we can have a community to plan and collaborate on projects?

**Comment by nahidalam** [comment]
Good idea. I would love to contribute but not sure where to start. Discord/Slack group might help

**Comment by gasvn** [comment]
Cool, I am new to Discord, let me figure out how to use it.
